app/src/main/java/com/architect/coders/mu8/remote/RetrofitService.kt

```diff
@@ -0,0 +1,24 @@
+package com.architect.coders.mu8.remote
+
+import com.architect.coders.mu8.remote.model.getallComic.GetAllComicsDTO
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/**
+ * All  marvel API Calls requires {@Link TS}, {@Link API_KEY}, and {@Link hash}
+ */
+interface RetrofitService {
+    companion object {
```

Comment:
Place the companion object at the end of the class as the [Kotlin style guidelines](https://kotlinlang.org/docs/reference/coding-conventions.html#class-layout) state.